packages/react/src/components/Authenticator/shared/UserNameAlias.tsx

```diff
@@ -31,37 +30,33 @@ export function UserNameAlias(props: UserNameAliasProps) {
     isPhoneAlias &&
       send({
         type: 'CHANGE',
-        data: { name: 'country_code', value: actorContext.country_code },
+        data: { name: 'country_code', value: country_code },
       });
   }, []);
 
-  return (
-    <>
-      {isPhoneAlias && (
-        <SelectField
-          name="country_code"
-          label="country code"
-          labelHidden={true}
-          defaultValue={actorContext.country_code}
-        >
-          {countryDialCodes.map((dialCode) => (
-            <option key={dialCode} value={dialCode}>
-              {dialCode}
-            </option>
-          ))}
-        </SelectField>
-      )}
-      <TextField
-        type={type}
-        name={alias ?? 'username'}
-        required
-        placeholder={i18nLabel}
-        label={label}
-        labelHidden={true}
-        autoComplete="username"
-        errorMessage={error}
-        {...attrs}
-      />
-    </>
+  return isPhoneAlias ? (
+    <PhoneNumberField
```

Comment:
Can you to add `autoComplete="username"` here instead of in the primitive? I'm guessing this is the use case where users use their phone number as their username?